pandas/plotting/_core.py

```diff
@@ -875,6 +883,14 @@ def _get_call_args(backend_name, data, args, kwargs):
                 "expected Series or DataFrame"
             )
 
+        if "sort_columns" in args + tuple(kwargs.keys()):
+            warnings.warn(
+                "`sort_columns` is deprecated and will be removed in a future "
+                "version.",
+                DeprecationWarning,
```

Comment:
We need a FutureWarning here

Reply:
I was a bit unsure about this, as I had seen both kinds of warnings in other commits.

For future reference, could you please elaborate on when do you use `FutureWarning` and `DeprecationWarning` here?

Reply:
DeprecationWarning is used if we want to make the user aware that we plan to remove/change this sometime in the future, but we will upgrade to a FutureWarning before that. It is a way to get some feedback too.

FutureWarning is used if we are certain that this will happen at the next major release.

i think we are using FutureWarnings almost everywhere. DeprecationWarnings are used for bigger changes, like changing the default dtype of an empty series

Reply:
Thanks for the explanation - that was very helpful!